### ISSUE TITLE
Use github.com/gopacket/gopacket

### ISSUE THREAD

**Issue by chancez** [x-way/pktdump#33]
Wondering if you would be willing to take contributions to switch to github.com/gopacket/gopacket. I was reviewing https://github.com/mozillazg/ptcpdump and noticed their fork of the repo https://github.com/x-way/pktdump/compare/master...mozillazg:pktdump:master and figured it might make sense to get the changes upstreamed, otherwise I'll just fork their fork, for my own project, but it would be nice to avoid so many forks for such simple changes.

**Comment by x-way** [comment]
Yes, switching to gopacket/gopacket makes sense. Can you send me a PR? I'll happily review and integrate it 👍

**Comment by chancez** [comment]
@x-way sure I'll try and do that sometime this week if I can find time.
